aten/src/ATen/native/mps/operations/Copy.mm

```diff
@@ -15,9 +15,17 @@
 namespace at {
 namespace native {
 
-MPSGraphTensor* chainViewOperation(MPSGraph* mpsGraph, IntArrayRef size,
-                             IntArrayRef stride, int64_t storage_offset,
-                             MPSGraphTensor* inputTensor, const Tensor& self) {
+MPSGraphTensor* chainViewOperation(
```

Comment:
I'm not sure why you ever need to chain views?
The data_ptr/size/stride/offset is enough to do any view. Chain views gives no benefit.

Reply:
True, this is just a name, it doesn't need any chaining. We will change the name in next commit.